app/(auth)/uuid.ts

```diff
@@ -1 +1 @@
-export const cookieName = 'user_uuid'
+export const USER_UUID_COOKIE_NAME = 'user_uuid'
```

Comment:
Do you think it's better to make `constants` file so that we can add any constants variable related  auth in this file in the future not only UUID? I think we should categorize as much as we can to avoid so many files or directory

Reply:
Make sense, actually I wondered too... 
renamed this to `constants.ts`